### ISSUE TITLE
Fatal Exception: NSInternalInconsistencyException – Multiplier is not finite! That's illegal. multiplier:nan

### ISSUE THREAD

**Issue by ldiqual** [Appboy/appboy-ios-sdk#102]
* [x] I have checked [Appboy's documentation](https://www.appboy.com/documentation/iOS/) and am still experiencing the issue.

# Report

## Describe your environment.

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | ios 
 Platform Version        | 10.0
 SDK Version             | 2.24.3
 Integration Method      | cocoapods
 Xcode Version           | 8.2
 Repro rate              | 100%
 
## What did you do?

Tried to show two in app notifications

## What did you expect to happen?

One in-app shows then the other. Or the second one replaces the other.

## What happened instead?

Got a crash:

```
Fatal Exception: NSInternalInconsistencyException
Multiplier is not finite! That's illegal. multiplier:nan
```

http://crashes.to/s/b72e9863eb9

Stacktrace is incomplete on web, but in the fabric iOS app here's what I get: 

![img_8211](https://cloud.githubusercontent.com/assets/941453/23349819/dda05140-fc6a-11e6-9688-397427c758f6.PNG)
![img_8210](https://cloud.githubusercontent.com/assets/941453/23349818/dd9e10b0-fc6a-11e6-9d1b-d4762ff19160.PNG)
![img_8209](https://cloud.githubusercontent.com/assets/941453/23349817/dd9de982-fc6a-11e6-8884-820f41d445c7.PNG)
![img_8208](https://cloud.githubusercontent.com/assets/941453/23349816/dd9c4136-fc6a-11e6-9aeb-748c7623ea96.PNG)

## Steps to reproduce

We have an in-app setup. It shows 15 seconds after a session starts and is set to send AGAIN after 3 days when a user has their next session (if they still qualify based on criteria).
When it attempts to show AGAIN, it crashes with the log I sent above. I was able to reproduce by targeting myself with a duplicate of the in-app (I receive the first one fine) and then setting it to resend after just 1 minute. After each minute, once I refresh, it crashes without showing the in-app.

## Side note

We couldn't find a similar issue or changelog line related to this crash, which is why we're assuming it's new. We're publishing tomorrow a new version of the app that contains the latest version of the SDK and we'll be able to confirm whether the crash still occurs or not.

**Comment by Wenzhi** [comment]
Hey @ldiqual,

Thank you for letting us know about this issue! We learned about the issue and released a patch in SDK version 2.24.4, which prevents the app from crashing because of the error. In version 2.26.0, which adopts the new SDWebImage 4.0 framework, we solved the issue from the root.

We believe that what happened was the images for triggered in-app messages were only stored in the cache by SDWebImage and got cleaned up after the first display. When the SDK tried to fetch the same image again, the returned image was nil. 

Thanks again for your detailed issue report :) Please let us know if you see the same error in the latest SDK.

Best,
Wenzhi

**Comment by ldiqual** [comment]
@Wenzhi Awesome! Thanks for following up, I'll let you know if we notice issues with the latest version. Have a great day.
